website/src/components/SearchPage/DownloadDialog/DownloadForm.tsx

```diff
@@ -114,19 +117,30 @@ export const DownloadForm: FC<DownloadFormProps> = ({ referenceGenomesSequenceNa
                     { label: <>Metadata</> },
                     {
                         label: <>Raw nucleotide sequences</>,
-                        subOptions: isMultiSegmented ? (
+                        subOptions: (
                             <div className='px-8'>
-                                <DropdownOptionBlock
-                                    name='unalignedNucleotideSequences'
-                                    options={referenceGenomesSequenceNames.nucleotideSequences.map((segment) => ({
-                                        label: <>{segment}</>,
-                                    }))}
-                                    selected={unalignedNucleotideSequence}
-                                    onSelect={setUnalignedNucleotideSequence}
+                                {isMultiSegmented ? (
+                                    <DropdownOptionBlock
+                                        name='unalignedNucleotideSequences'
+                                        options={referenceGenomesSequenceNames.nucleotideSequences.map((segment) => ({
+                                            label: <>{segment}</>,
+                                        }))}
+                                        selected={unalignedNucleotideSequence}
+                                        onSelect={setUnalignedNucleotideSequence}
+                                        disabled={dataType !== 1}
+                                    />
+                                ) : undefined}
+                                <RadioOptionBlock
+                                    name='richFastaHeaders'
+                                    title='FASTA header style'
+                                    options={[{ label: <>Accession only</> }, { label: <>Accession and metadata</> }]}
```

Comment:
I would maybe make the option Accession or Display Name (as we do label the display name as display name on the sequence view pages and which metadata fields are added is ambiguous)